bigframes/dataframe.py

```diff
@@ -3106,6 +3107,12 @@ def to_gbq(
                 + constants.DEFAULT_EXPIRATION,
             )
 
+        if len(labels) != 0:
+            client = bigquery.Client()
```

Comment:
Instead of create a new client every time, we should reuse the df.session.bqclient.

Reply:
done!